### PR TITLE
Verify script updates

### LINKED ISSUE #508: Updates based on verify script from Travis run
Hey guys,

Been working on updating the verify.rb script.

Heres some findings from a travis output thats using adult section as well, which is not yet on the main branch.
https://travis-ci.org/kenman345/acceptbitcoincash/jobs/335166740#L943

I am making a branch with the changes I got so far so we have a branch to work off of and will reference this issue in the first commit. 

Here is the breakdown of the effort to revise parts of this script and adjust accordingly:
- [x] Add tags to schema yml and fix bad tags in yml
- [x] Run a tool through the images to ensure they are smaller than expected.
- [x] alphabetize the yml files
- [x] revise pattern matching

### PR DESCRIPTION
**Pull request by kenman345**
Updates to verify script allow us to verify that the yml fields are filled properly, that all the tags are expected and not mislabelled, and that we maintain ordering of those yml files alphabetically. 

The changes made to the script will output the list of sites in the correct order when it identifies that its currently not in the right order and fails the build. I think this will give us much better results and keep things running smoothly.

I also pushed changes to the **[acceptbitcoincash/acceptbitcoincash:feature-xxx-page](/acceptbitcoincash/acceptbitcoincash/tree/feature-xxx-page)** so that it will not fail under these new changes, but these wont take effect until that branch is a part of a PR or is rebased after merging this pull request

Closes #508

### CODE REVIEW

**Comment by kenman345** [comment]
Forgot to mention, closes #502

File: verify.rb [hunk 1]
```diff
@@ -3,28 +3,28 @@
...
-@tfa_tags = {
-  # YAML tags for TFA Yes
-  true => %w[email hardware software sms phone doc],
-  # YAML tags for TFA No
-  false => %w[status twitter facebook email_address lang]
-}.freeze
-
 # Image max size (in bytes)
 @img_max_size = 2500
+#@img_max_size = 3000
```
Comment (joshmcintyre) [inline verify.rb]: Any particular reason for the bump in max image size?

File: verify.rb [hunk 2]
```diff
@@ -3,28 +3,28 @@
...
 
 # Image dimensions
 @img_dimensions = [32, 32]
+@img_lg_dimensions = [48, 48]
```
Comment (joshmcintyre) [inline verify.rb]: Same here, just curious why the bump. Don't think it will hurt anything repo-size wise.

File: websites_schema.yml [hunk 3]
```diff
@@ -12,21 +12,22 @@ mapping:
             unique: yes
           "url":
             type: str
-            pattern: /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/
+            pattern: /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,8}\b([-a-zA-Z0-9@:%_\+.~#?&\/\/=]*)/
```
Comment (joshmcintyre) [inline websites_schema.yml]: For future reference, Dan's Tools Regexpal is great for checking regex correctness: https://www.regexpal.com/

May be a good idea to run some strings through this when modifying/updating patterns.

File: verify.rb (hunk 2 above)
Comment (kenman345) [reply verify.rb]: We plan to use 48x48 for charity page. The page is gonna be different so I added the size to validation

File: verify.rb (hunk 1 above)
Comment (kenman345) [reply verify.rb]: Thats commented. I used that during my initial tests to see the rest of the verifications.

The hashtag means its commented

File: websites_schema.yml (hunk 3 above)
Comment (kenman345) [reply websites_schema.yml]: thanks for that. we had a .software url that was failing

**Comment by kenman345** [comment]
@t0p0lin0 I know you're uber busy but I would like a second approval on this because its not just content changes, its our process

**Pull request merged**